dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -445,6 +445,7 @@ Simulation::compileModels() {
           modelicaModelsDirsAbsolute,
           modelicaModelsExtension,
           additionalHeaderFiles,
+          true,
```

Comment:
please define a const boolean to improve readability: const bool rmModels = true